src/System.Runtime.Numerics/src/System/Numerics/Complex.cs

```diff
@@ -185,138 +135,73 @@ public static Double Abs(Complex value)
-        }
-        public static implicit operator Complex(Byte value)
-        {
-            return (new Complex(value, 0.0));
-        }
-        public static implicit operator Complex(Single value)
-        {
-            return (new Complex(value, 0.0));
-        }
-        public static implicit operator Complex(Double value)
-        {
-            return (new Complex(value, 0.0));
-        }
-        public static explicit operator Complex(BigInteger value)
-        {
-            return (new Complex((Double)value, 0.0));
-        }
-        public static explicit operator Complex(Decimal value)
+        public bool Equals(Complex value)
         {
-            return (new Complex((Double)value, 0.0));
+            return _real.Equals(value._real) && _imaginary.Equals(value._imaginary);
         }
 
-
-        // --------------SECTION: Formattig/Parsing options  -------------- //
-
-        public override String ToString()
+        public override int GetHashCode()
         {
-            return (String.Format(CultureInfo.CurrentCulture, "({0}, {1})", _real, _imaginary));
+            int n1 = 99999997;
+            int realHash = _real.GetHashCode() % n1;
+            int imaginaryHash = _imaginary.GetHashCode();
+            int finalHash = realHash ^ imaginaryHash;
+            return finalHash;
         }
-
-        public String ToString(String format)
+        
+        public override string ToString()
         {
-            return (String.Format(CultureInfo.CurrentCulture, "({0}, {1})", _real.ToString(format, CultureInfo.CurrentCulture), _imaginary.ToString(format, CultureInfo.CurrentCulture)));
+            return ToString(string.Empty);
         }
 
-        public String ToString(IFormatProvider provider)
+        public string ToString(string format)
         {
-            return (String.Format(provider, "({0}, {1})", _real, _imaginary));
+            return ToString(format, CultureInfo.CurrentCulture);
         }
 
-        public String ToString(String format, IFormatProvider provider)
+        public string ToString(IFormatProvider provider)
         {
-            return (String.Format(provider, "({0}, {1})", _real.ToString(format, provider), _imaginary.ToString(format, provider)));
+            return ToString(string.Empty, provider);
         }
 
-
-        public override Int32 GetHashCode()
+        public string ToString(string format, IFormatProvider provider)
         {
-            Int32 n1 = 99999997;
-            Int32 hash_real = _real.GetHashCode() % n1;
-            Int32 hash_imaginary = _imaginary.GetHashCode();
-            Int32 final_hashcode = hash_real ^ hash_imaginary;
-            return (final_hashcode);
+            return string.Format(provider, "({0}, {1})", _real.ToString(format, provider), _imaginary.ToString(format, provider));
         }
-
-
-
-        // --------------SECTION: Trigonometric operations (methods implementing ITrigonometric)  -------------- //
-
+        
```

Comment:
Please revert these changes to ToString.  From my perspective, these changes make it harder to understand the behavior of most of the overloads, and it can actually add some cost.